content/devdocs/references/p2p_networking.md

```diff
@@ -250,7 +357,96 @@ The following annotated hexdump shows a `GIVT` message.  (The
 message header has been omitted.)
 
 {{ <highlight text> }}
-TODO : GIVT message hex dump
+GiveTxnsMessage:
+```
+82 02 00 00 00 00 00 00 02 00 00 00 88 13 00 00 
+7b 83 d5 44 cc c2 23 c0 57 d2 bf 80 d3 f2 a3 29 
+82 c3 2c 3c 0d b8 e2 67 48 20 da 50 64 78 3f b0 
+97 02 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 02 00 00 00 83 d5 44 cc c2 
+23 c0 57 d2 bf 80 d3 f2 a3 29 82 c3 2c 3c 0d b8 
+e2 67 48 20 da 50 64 78 3f b0 97 83 d5 44 cc c2 
+23 c0 57 d2 bf 80 d3 f2 a3 29 82 c3 2c 3c 0d b8 
+e2 67 48 20 da 50 64 78 3f b0 97 02 00 00 00 00 
+01 17 9d 05 40 93 65 99 ec ac e9 50 e1 c4 4e f2 
+64 63 42 78 0c 00 00 00 00 00 00 00 22 00 00 00 
+00 00 00 00 00 dc 5e f1 cf 10 46 c3 e3 59 9e e0 
+ad 91 84 4b 25 4d 21 d2 05 38 00 00 00 00 00 00 
+00 4e 00 00 00 00 00 00 00 88 13 00 00 7b 83 d5 
+44 cc c2 23 c0 57 d2 bf 80 d3 f2 a3 29 82 c3 2c 
+3c 0d b8 e2 67 48 20 da 50 64 78 3f b0 97 02 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
+00 00 00 00 02 00 00 00 83 d5 44 cc c2 23 c0 57 
+d2 bf 80 d3 f2 a3 29 82 c3 2c 3c 0d b8 e2 67 48 
+20 da 50 64 78 3f b0 97 83 d5 44 cc c2 23 c0 57 
+d2 bf 80 d3 f2 a3 29 82 c3 2c 3c 0d b8 e2 67 48 
+20 da 50 64 78 3f b0 97 02 00 00 00 00 78 cf d3 
+d9 82 fc 1c ec a4 2f 15 fc 10 5a 48 2b 82 35 25 
+76 09 00 00 00 00 00 00 00 0c 00 00 00 00 00 00 
+00 00 c3 be 85 8a b1 76 0c e4 05 5e 65 cc c3 a3 
+53 7e 75 bb 72 07 22 00 00 00 00 00 00 00 38 00 
+00 00 00 00 00 00 ................................. Full message
+------------------------------------------------------------------------
+0x0000 82 02 00 00 00 00 00 00 ........................... Prefix
```

Comment:
This is not quite like it should be . First four bytes are the total size of the buffer that follows . The other four bytes are the message `Prefix` ID (e.g.one of `INTR`, `GETB`, ...) . Please fix that